.travis.yml

```diff
@@ -22,6 +22,8 @@ matrix:
       env: WP_VERSION=3.7.11
     - php: 5.3
       env: WP_VERSION=3.7.11 DEPLOY_BRANCH=master
+    - php: 7.1
+      env: WP_VERSION=latest SAVE_COMMENTS_DISABLED=1
```

Comment:
The test tagged `@require-opcache-save-comments` can't be run with `opcache.save_comments` disabled - I'm not getting you.